L1Trigger/Phase2L1ParticleFlow/test/make_l1ct_patternFiles_cfg.py

```diff
@@ -1,3 +1,18 @@
+import argparse
+import sys
+
+# example: cmsRun L1Trigger/Phase2L1ParticleFlow/test/make_l1ct_patternFiles_cfg.py -- --dumpFilesON
+
+parser = argparse.ArgumentParser(prog=sys.argv[0], description='Tune parameters')
+
+parser.add_argument("--dumpFilesON", help="switch on dump file production", action="store_true", default=False)
+parser.add_argument("--Layer1PatternsOFF", help="switch on Layer-1 pattern file production", action="store_true", default=False)
```

Comment:
I think it would make more sense to have them called `--dumpFiles`  and `--noLayer1Patterns` 

However, maye the best is just to merge dumpFile production and pattern writing configs into one, and have it produce all (I believe there's not much saving in switching things off except for some clutter in your working directory)

Reply:
Well, this config now does everything: we could rename it and drop the others. 
The idea is that you are not forced to use the flags since there are defaults. Still, the job produces A LOT of data, and switching some off could be handy: you get the same functionality of having multiple cfg without having to maintain them. This is also true when running  CI.

If you are ok with the approach, we could agree on better names and sensible defaults.